exercicios/para-casa/resolucao/larissa-gnandt/pseudoclasses.js

```diff
@@ -0,0 +1,108 @@
+const {
+  imprimeVacina,
+  imprimeConsultas,
+  imprimeProcedimento,
+} = require("./utils");
+
+function Gato(nome, idade, cor, castrado, historico) {
+  this.nome = nome;
+  this.idade = idade;
+  this.cor = cor;
+  this.castrado = castrado;
+  this.historico = historico;
+
+  this.vacinar = function (vacina) {
```

Comment:
Dessa forma funciona legal, porem dava tambem para colocar o metodo no prototype,  pra nao repetir sempre o mesmo metodo em todas as pseudoclasses...tipo assim:
Gato.prototype.vacinar = function (vacina) {
  this.historico.vacinas.push(vacina);
  imprimeVacina(this.nome, vacina);
}

Reply:
Oie Amanda querida, hahaha eu vi isso no código que fui revisar, como poderia estar melhor esta parte kkkkk Obrigada pela sugestão :)